### PR TITLE
Package vscoq-language-server.2.0.0+coq8.18

### PR DESCRIPTION
**Pull request by vscoqbot**
### `vscoq-language-server.2.0.0+coq8.18`
VSCoq language server
LSP based language server for Coq and its VSCoq user interface

---
* Homepage: https://github.com/coq-community/vscoq
* Source repo: git+https://github.com/coq-community/vscoq
* Bug tracker: https://github.com/coq-community/vscoq/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

File: packages/vscoq-language-server/vscoq-language-server.2.0.0+coq8.18/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+license: "MIT"
+homepage: "https://github.com/coq-community/vscoq"
+bug-reports: "https://github.com/coq-community/vscoq/issues"
+dev-repo: "git+https://github.com/coq-community/vscoq"
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+depends: [
+  "ocaml" { >= "4.13.1" }
```
Comment (palmskog) [inline packages/vscoq-language-server/vscoq-language-server.2.0.0+coq8.18/opam:13]:
```suggestion
  "ocaml" { >= "4.13.1" }
  "dune" { >= "3.2" }
```
Linter complains about this. The 3.2 is via the `dune-project` file.

**Review by rtetley — APPROVED** [review]

File: packages/vscoq-language-server/vscoq-language-server.2.0.0+coq8.18/opam [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "dune" { >= "3.2" }
+  "coq-core" { >= "8.18" < "8.19" }
+  "coq-stdlib" { >= "8.18" < "8.19" }
+  "yojson"
```
Comment (gares) [inline packages/vscoq-language-server/vscoq-language-server.2.0.0+coq8.18/opam:17]:
```suggestion
  "yojson"
  "jsonrpc" { >= "1.16" }
```

**Comment by gares** [comment]
CI looks good now.
The two failures on 32 bit systems will be addressed upstream with another realease I guess.

File: packages/vscoq-language-server/vscoq-language-server.2.0.0+coq8.18/opam [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  "jsonrpc" { >= "1.16" }
```
Comment (gares) [inline packages/vscoq-language-server/vscoq-language-server.2.0.0+coq8.18/opam:18]:
```suggestion
  "jsonrpc" { >= "1.12" }
```

Comment (gares) [reply packages/vscoq-language-server/vscoq-language-server.2.0.0+coq8.18/opam:18]: This should unlock more ocaml version

**Comment by gares** [comment]
Damn, jsonrpc 1.12  does not work, I've the impression there are two `Pp` modules in scope.
So we should raise the lower bound apparently, but I could not really figure out the correct value.
We could try 1.15, according to the changelog it should not be too picky.

File: packages/vscoq-language-server/vscoq-language-server.2.0.0+coq8.18/opam [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+depends: [
+  "ocaml" { >= "4.13.1" }
+  "dune" { >= "3.2" }
+  "coq-core" { >= "8.18" < "8.19" }
+  "coq-stdlib" { >= "8.18" < "8.19" }
+  "yojson"
+  "jsonrpc" { >= "1.12" }
```
Comment (gares) [inline packages/vscoq-language-server/vscoq-language-server.2.0.0+coq8.18/opam:18]:
```suggestion
  "jsonrpc" { >= "1.15" }
```

**Comment by rtetley** [comment]
What can we do to get this PR merged ? The remaining errors can only be fixed upstream

**Comment by gares** [comment]
Yes please consider this for merge. An upcoming release will relax some constraint, but in the meantime this one will unblock some users.

**Comment by maximedenes** [comment]
@rtetley Should this be closed in favor of https://github.com/ocaml/opam-repository/pull/24486 ?

**Comment by rtetley** [comment]
> @rtetley Should this be closed in favor of https://github.com/ocaml/opam-repository/pull/24486 ?

Yup

**Pull request closed without merging**